packages/devtools_app/lib/main.dart

```diff
@@ -48,6 +48,14 @@ void main() {
         return;
       }
 
+      // Show the Q3 DevTools survey.
+      // TODO(kenz): show this survey once per user. Turn it off until we add
```

Comment:
showing more than once per user could be ok we should just not show it again after they take an action to dismiss it or after we have shown it 5 times or after the date is after October 30th(?)

Reply:
updated comment to reflect show conditions